SMLHelper/mod_BelowZero.json

```diff
@@ -2,9 +2,8 @@
   "Id": "SMLHelper",
   "DisplayName": "Modding Helper (SMLHelper)",
   "Author": "The QModManager Team",
```

Comment:
Make sure that the author is the same.
maybe make it `The SMLHelper Dev Team`

Reply:
https://github.com/SubnauticaModding/SMLHelper/pull/126/commits/3aa543f2436da65641792e8006401aa06c4f8179